packages/corvid-cli/src/commands/login.js

```diff
@@ -92,7 +92,9 @@ async function loginCommand(spinner, args = {}) {
       ? messages.filter(({ msg }) => msg === "authCookie")
       : [];
     const authCookie = _.get(cookieMessages, [0, "cookie"]);
-    await sessionData.set({ uuid: parseSessionCookie(authCookie).userGuid });
+    const userGuid = parseSessionCookie(authCookie).userGuid;
+    sentry.setUserId(userGuid);
```

Comment:
the "setUserId" method should be exposed on the logger instance.
we shouldn't expose sentry as is outside of the logger module